jsk_perception/node_scripts/draw_classification_result.py

```diff
@@ -61,10 +57,14 @@ def _draw(self, cls_msg, imgmsg):
             scale = min(scale_h, scale_w)
             (text_w, text_h), baseline = cv2.getTextSize(
                 label_name, cv2.FONT_HERSHEY_SIMPLEX, scale, 1)
+            if LooseVersion(cv2.__version__).version[0] == 3:
+                line_type = cv2.LINE_AA
+            else:  # OpenCV2
```

Comment:
I prefer
```
if version < 3.0
  line_type = cv2.CV_AA
else # for opencv >= 3
  line_type  = cv2.LINE_AA
```
because this code has more chance to survive when opencv is updated to 4

Reply:
thank you. I updated